searx/engines/loc.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+
+ Library of Congress : images from Prints and Photographs Online Catalog
+
+"""
+
+from json import loads
+from urllib.parse import urlencode
+
+
+about = {
+    "website": 'https://www.loc.gov/pictures/',
+    "wikidata_id": 'Q131454',
+    "official_api_documentation": 'https://www.loc.gov/pictures/api',
+    "use_official_api": True,
+    "require_api_key": False,
+    "results": 'JSON',
+}
+
+categories = ['images']
+
+paging = True
+
+base_url = 'https://loc.gov/pictures/search/?'
+search_string = "&sp={page}&{query}&fo=json"
+
+
+def request(query, params):
+
+    search_path = search_string.format(
+        query=urlencode({'q': query}),
+        page=params['pageno'])
+
+    params['url'] = base_url + search_path
+
+    return params
+
+
+def response(resp):
+    results = []
+
+    json_data = loads(resp.text)
+
+    for result in json_data['results']:
+        results.append({'url': result['links']['item'],
```

Comment:
Aligned with opening delimiter is PEP-8 conform, but I wanted to ask if we  can we use hanging indents which is most often better to read:

```python
    for result in json_data['results']:
        results.append({
            'url': result['links']['item'],
            'title': result['title'],
            'img_src': result['image']['thumb'],
            'template': 'images.html'
        })
```

Reply:
I agree, it's easier to read.